admin/views/pupil-register/pupils-list.ejs

```diff
@@ -59,18 +59,12 @@
                     >
                         <div class="highlight-wrapper">
                             <% if (isFeatureEnabled('pupilEdit') && !availabilityData.hdfSubmitted && !availabilityData.checkWindowClosed) { %>
-                                <a class="bold-small truncate" href="/pupil-register/pupil/edit/<%= p.urlSlug %>" >
-                                    <%= `${p.fullName}` %>
-                                </a>
+                                <a class="bold-small name-text-wrap" href="/pupil-register/pupil/edit/<%= p.urlSlug %>" ><%= `${p.fullName}` %></a>
```

Comment:
You don't need to convert the variable to a string before outputting it, just the variable will do.  Am I missing something?

Reply:
This has been there forever.
I only removed the spaces but I didn't notice it was stringified, it's updated now, good catch!